src/eosjs-api.ts

```diff
@@ -317,14 +351,101 @@ export class Api {
             });
         }
         if (broadcast) {
-            if (compression) {
-                return this.pushCompressedSignedTransaction(pushTransactionArgs);
+            const result = await this.rpc.send_transaction(pushTransactionArgs);
```

Comment:
What happened to `pushCompressedSignedTransaction`?

Reply:
Reverted that change